client/src/components/helpers/spacer.tsx

```diff
@@ -10,13 +10,15 @@ const Padding = Object.freeze({
 type PaddingKeys = keyof typeof Padding;
 interface SpacerProps {
   size: PaddingKeys;
+  children?: React.ReactNode;
 }
 
-const Spacer = ({ size }: SpacerProps): JSX.Element => (
-  <div
-    className='spacer'
-    style={{ padding: `${Padding[size]}px 0`, height: '1px' }}
-  />
+const Spacer = ({ size, children }: SpacerProps): JSX.Element => (
+  <div className='spacer' style={{ paddingBlock: `${Padding[size]}px` }}>
+    {children && (
+      <div style={{ paddingBlock: `${Padding[size]}px` }}>{children}</div>
+    )}
```

Comment:
although this gives the desired looks, this is a `if` statement and another `div` without the need to that users have to pay for that load which isn't necessary, you can update the values in `Padding` instead.

```suggestion
    {children}
```

Reply:
That's correct!! It could be done in the previous step itself. It works!